### PR TITLE
refactor: Improve performance, seo, accessibility and security

### LINKED ISSUE #45: home.html: Missing quotation marks in alt attribute in intro image
In `home.html` :

``

Once the site is generated :

`Jekyll`

Solved in #44 with this solution :

`{{ page.title }}`

### PR DESCRIPTION
**Pull request by Balaiseee**
Performance improvement
- style.scss & _fonts.scss: Self host font
- teams.html: Set width and height for img
- header.html: Set width and height for img
- home.html: Set width and height for img

SEO improvement
- about.md: Add description label
- contact.md: Add description label
- services.md: Add description label
- team.md: Add description label
- home.html: Fix #45

Accessibility improvement
- footer.html: Fix HTML hierarchy problem (replace `h3` by `h2`)
- team.html: Increase contrast ratio (add `$primary-dark` variable in style.scss and set `a` color to `$primary-dark` in _type.scss)

Security improvement
- teams.html: Fix LinkedIn link (add `rel="noreferrer"` attribute)

For all the pages (except subcategories of services and team, coming soon):

- 100/100 Google Lighthouse speed score
- 100/100 Google Lighthouse seo score
- 100/100 Google Lighthouse accessibility score
- 100/100 Google Lighthouse best practices score

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <title>{% if page.title %}{{page.title}}{% else %}{{ site.title | escape }}{% endif %}</title>
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <link rel="icon" type="image/png" href="{{ '/images/favicon-32x32.svg' | relative_url }}">
-  <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700&display=swap" rel="stylesheet">
+  <link rel="preload" href="{{ '/assets/fonts/playfair-display.woff2' | relative_url }}" as="font" type="font/woff2" crossorigin>
```
Comment (JugglerX) [inline _layouts/default.html:9]: I'm skeptical that self hosting the font will make things faster. And this change makes it harder for people to change the fonts. 

I suggest we continue to use Google Fonts cdn as the default, but we include the code to self host the font and have it commented out.

File: _team/bill-mcdonald.md [hunk 2]
```diff
@@ -3,6 +3,8 @@ title: "Bill McDonald"
 date: 2018-11-19T10:47:58+10:00
 draft: false
 image: "images/team/nonsap-visuals-kMJp7620W6U-unsplash.jpg"
+image_width: 60
```
Comment (JugglerX) [inline _team/bill-mcdonald.md:6]: Adding new values for width and height in the front-matter is a bit verbose.

For the team page, I'd probably just set the width and height of 60px in the HTML. It's a pretty decent default.

**Review by JugglerX — CHANGES_REQUESTED** [review]
@Balaiseee some really great changes! I appreciate the PR. I've left a few small comments about the fonts and the team image. We need to keep in mind that this is a theme that many other people will use. And we need to strike the balance between it being "perfect" without adding to much complexity.

File: _layouts/default.html (hunk 1 above)
Comment (Balaiseee) [reply _layouts/default.html]: Done!

File: _team/bill-mcdonald.md (hunk 2 above)
Comment (Balaiseee) [reply _team/bill-mcdonald.md]: Done!

**Review by JugglerX — APPROVED** [review]

**Comment by Emkann** [comment]
@

**Pull request merged**